src/defaultsettings.cpp

```diff
@@ -69,6 +69,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("enable_build_where_you_stand", "false" );
 	settings->setDefault("3d_mode", "none");
 	settings->setDefault("3d_paralax_strength", "0.025");
+	settings->setDefault("3d_convergence_distance", "1.95");
```

Comment:
Yeah, it has to be documented in builtin/defaultsettings.txt, minetest.conf.example isn't required (and will be overwritten anyway the next time one automatically updates those files).